packages/containers/src/ActionButton/__snapshots__/ActionButton.test.js.snap

```diff
@@ -3,7 +3,15 @@
 exports[`CMF(Container(ActionButton)) should render 1`] = `
 <withI18nextTranslation(ActionButton)
   actionId="menu:article"
+  available={true}
+  bsStyle="default"
+  disabled={false}
   extra="foo"
+  inProgress={false}
+  label="click"
+  loading={false}
   onClick={[Function]}
+  t={[Function]}
+  tooltipPlacement="top"
```

Comment:
All the snapshots changes are due to default props. They now appear with the new version of jest